MyBus/MyBus/ViewController.swift

```diff
@@ -19,6 +19,14 @@ class ViewController: UIViewController {
         // Do any additional setup after loading the view, typically from a nib.
         
         mapView.userTrackingMode = .Follow
+        mapView.delegate = self
+    }
+    
+    func mapView(
+        mapView: MGLMapView,
+        didUpdateUserLocation userLocation: MGLUserLocation?) {
```

Comment:
Can we have the whole method signature written on a single line as
`func mapView(mapView: MGLMapView, didUpdateUserLocation userLocation: MGLUserLocation?)`